src/js/components/menu/MainMenu.vue

```diff
@@ -144,7 +144,9 @@ export default {
   props: {
     open: Boolean,
     keepOpen: Boolean,
-    corpus: {},
+    corpus: {
+      required: true,
+    },
```

Comment:
I'm not settled in my mind on how much we want to do in prop validation (e.g., passing in a class that we're expected, required/not, etc). There's a runtime penalty for stuff like that and I'm not sure how much we gain. In any case, something to consider as we go.

interestingly, discovered why you needed the bind `:propName="42"` for the number: https://vuejs.org/v2/guide/components-props.html#Passing-a-Number

Reply:
@Bronson-Brown-deVost : I just mean that there's a penalty to do required and type checks since Vue has to run extra code. I don't know if it's huge ... I've heard that React has a way to do this for dev only but disable for production ... we could look into that. For the moment, I think they're nice to have and catch errors I make.